src/ape/api/accounts.py

```diff
@@ -203,9 +286,28 @@ def remove(self, account: AccountAPI):
         self.__delitem__(account.address)
 
     def __delitem__(self, address: AddressType):
+        """
+        Delete an account.
+        Must be overriden or else raises ``NotImplementedError``.
+
+        Args:
+            address: (address :class:`~ape.types.AddressType`):
+            The address of the account to delete.
```

Comment:
Some of this should start on the line above and then this second line should be indented a little bit

Reply:
black . had an issue with that. i broke it up like that.

is it okay. Or how should i break it up?

Reply:
You should be able to at least indent it a little bit